data_validation/partition_builder.py

```diff
@@ -320,27 +341,24 @@ def _add_partition_filters(
 
             yaml_configs_list[ind] = {
                 "target_folder_name": config_manager.full_source_table,
-                "partitions": [],
+                "yaml_files": [],
             }
-            for pos in range(len(filter_list[0])):
-                filter_dict = {
-                    "type": "custom",
-                    "source": filter_list[0][pos],
-                    "target": filter_list[1][pos],
-                }
-                # Append partition new filter
-                config_manager.filters.append(filter_dict)
 
+            # Create a list of lists chunked by partitions per file
+            source_filters_list = list_to_sublists(
```

Comment:
Do we need to check that len(source_filters_list) == len(target_filters_list)? Or can they have different lengths